Plugins/UnityServerPlugin/StreamingUnityServerPlugin.cpp

```diff
@@ -61,27 +67,26 @@
 using namespace Microsoft::WRL;
 using namespace StreamingToolkit;
 
-DEFINE_GUID(IID_Texture2D, 0x6f15aaf2, 0xd208, 0x4e89, 0x9a, 0xb4, 0x48, 0x95, 0x35, 0xd3, 0x4f, 0x9c);
-
-static IUnityInterfaces* s_UnityInterfaces = nullptr;
-static IUnityGraphics* s_Graphics = nullptr;
-static UnityGfxRenderer s_DeviceType = kUnityGfxRendererNull;
-static ComPtr<ID3D11Device> s_Device;
-static ComPtr<ID3D11DeviceContext> s_Context;
+static IUnityInterfaces*			s_UnityInterfaces		= nullptr;

```

Comment:
nit: fwiw, these spacing changes make things __way__ harder to read on github...i assume they look good in a local editor?

Reply:
github issue, looks fine in VS.